README.md

```diff
@@ -1,59 +1,76 @@
+- [Building a Curriculum for VWC](#building-a-curriculum-for-vwc)
+  * [Curriculum Structure](#curriculum-structure)
+    + [:factory: Folder Structure :factory:](#-factory--folder-structure--factory-)
+    + [Root level READMEs](#root-level-readmes)
+      - [Rationale](#rationale)
+      - [Objectives](#objectives)
+      - [Prerequisites](#prerequisites)
+      - [Syllabus](#syllabus)
+      - [Next Step](#next-step)
+  * [Build Learning Activities](#build-learning-activities)
+  * [Evaluate the Learners Performance](#evaluate-the-learners-performance)
+  * [Provide Feedback on Performance.](#provide-feedback-on-performance)
+
+<small><i><a href='http://ecotrust-canada.github.io/markdown-toc/'>Table of contents generated with markdown-toc</a></i></small>
+
 # Building a Curriculum for VWC
 Keep in mind that laying out a well thought-out and comprehensive curriculum that achieves our organizational goals is an iterative process. We seek to continually improve the effectiveness of our courses by improving learning outcomes. 
 
 Primary focus at the beginning is on writing learning objectives and an overall content structure to support those objectives. 
 
-## 1. Overall Objective
-This describes the intended outcome of the course from the learners perspective. 
-
-*Ex. By completing this course the learner will have all the essential skills to be hired as a Frontend Developer.*
-
-TODO - Provide a guide for how to write a solid overall objective. 
+## Curriculum Structure
 
-## 2. Course Prerequisites
```

Comment:
Removed numbers because it makes reordering more challenging in the future. Also added alot of these sections as sub sections under `Root level READMEs`